binaries/speed_benchmark_torch.cc

```diff
@@ -114,8 +124,7 @@ int main(int argc, char** argv) {
     "Example usage:\n"
     "./speed_benchmark_torch"
     " --model=<model_file>"
-    " --input_dims=\"1,3,224,224\""
-    " --input_type=float"
```

Comment:
So this is bc breaking change for speed_benchmark right? It breaks all the AIBench continuous run.

Reply:
This is just an update to the usage message.  The old args still work.